Makefile

```diff
@@ -126,12 +152,12 @@ govulncheck:
 
 INSTALL_PREFIX?=/usr/local/
 
-install: $(PREFIX)bin/$(BINNAME)
+install: $(PREFIX)bin/step
 	$Q mkdir -p $(INSTALL_PREFIX)bin/
-	$Q install $(PREFIX)bin/$(BINNAME) $(DESTDIR)$(INSTALL_PREFIX)bin/$(BINNAME)
+	$Q install $(PREFIX)bin/step $(DESTDIR)$(INSTALL_PREFIX)bin/step
 
 uninstall:
-	$Q rm -f $(DESTDIR)$(INSTALL_PREFIX)/bin/$(BINNAME)
+	$Q rm -f $(DESTDIR)$(INSTALL_PREFIX)/bin/step
```

Comment:
$BINNAME is missing.

And why do we use both `DESTDIR` and `INSTAL_PREFIX`. `DESTDIR` is the standard one that most GNU makefile supports.